tutorial/part1-creating-a-custom-theme/create-a-theme-package.md

```diff
@@ -59,6 +59,24 @@ And replace it with:
 },
 ```
 
+Since we will not be using the mars-theme installed on package.json, we should remove it from the dependencies list.
+
+```json
+// File: /package.json
+
+{
+  "dependencies": {
+    "@frontity/core": "^1.10.1",
+    "@frontity/html2react": "^1.6.1",
+    "@frontity/mars-theme": "./packages/mars-theme", // <-- Remove this line
+    "@frontity/tiny-router": "^1.3.2",
```

Comment:
Thanks @kokiebisu for this PR. I think this is only necessary if you delete `mars-theme` from the `packages` folder. If you leave the `mars-theme` folder in place then you can switch to the newly created theme without editing the `package.json` file. 

Please correct me if I'm wrong.

I understand the temptation to "tidy things up" by removing the unused theme - in which case, yes, you need to remove this line.

Reply:
Thanks for the reply. In the previous step, we have a step to remove @frontity/mars-theme. This may cause some misunderstandings. We may consider removing it or adding the step above just for clarity:)
![スクリーンショット 2021-02-17 午前9 08 57](https://user-images.githubusercontent.com/43525282/108240838-161efb80-7100-11eb-9c84-9dc7aa09c659.png)

Reply:
@juanmaguitar What do you think of the idea of adding this? We're not instructing people to remove the `mars-theme` directory, so should we suggest that they remove the line from `package.json` in case they do?

Reply:
@mburridge I agree with you. As we're not advising to remove the default `mars-theme` people shouldn't get this error.
Let's wait to see if this happens to more people before adding more things to be taken into account to the tutorial